src/pipeline.cxx

```diff
@@ -324,9 +327,9 @@ void pqxx::pipeline::obtain_dummy()
     do
     {
       m_num_waiting--;
-      auto const query{*m_issuedrange.first->second.get_query()};
+      auto const query{*m_issuedrange.first->second.query};
       result const res{m_trans.exec(query)};
-      m_issuedrange.first->second.set_result(res);
+      m_issuedrange.first->second.res = res;
```

Comment:
This seems odd.

There's no meaningful way to use std::move() on the result object here?

Reply:
Good catch.  This code predates move semantics by a good many years, so there will be opportunities for re-optimisation.  Copying results may be cheap, but not copying is cheaper.